### PR TITLE
Remove CONTENT_TYPE in kubemark hollow template

### PR DESCRIPTION
**Pull request by Jeffwan**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

This is a follow up PR of https://github.com/kubernetes/community/pull/5476
We want to explicitly pass arguments `--kube-api-content-type=$(CONTENT_TYPE)`. Otherwise, people will get confused about the value of `$(CONTENT_TYPE)`. This helps to have consistency across the kubemark tests. 

I tried to search all references but I can not find anywhere set this value.  I assume kubemark test didn't specify it. 
Reasonable value of `CONTENT_TYPE` could be 
- application/vnd.kubernetes.protobuf
- application/json

**Which issue(s) this PR fixes**:

Fixes #https://github.com/kubernetes/community/pull/5476#pullrequestreview-585240063

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
N/A
```

### CODE REVIEW

**Comment by Jeffwan** [comment]
@mm4tt  configmap is created here. 
https://github.com/kubernetes/kubernetes/blob/6389c13960cf1d37df5aebfdc5e520dbc4ea1787/test/kubemark/start-kubemark.sh#L92-L94

The problem is I can not find the value of `TEST_CLUSTER_API_CONTENT_TYPE`.  Do you know where this env is exported? 
I want to double check if CI/CD assigned value like `--kube-api-content-type=application/vnd.kubernetes.protobuf`.

File: test/kubemark/resources/hollow-node_template.yaml [hunk 1]
```diff
@@ -55,7 +55,7 @@ spec:
           "--morph=kubelet",
           "--name=$(NODE_NAME)",
           "--kubeconfig=/kubeconfig/kubelet.kubeconfig",
-          "$(CONTENT_TYPE)",
```
Comment (mm4tt) [inline test/kubemark/resources/hollow-node_template.yaml:58]: Let's remove it and remove it from the configmap as well. I'd like to go into a direction where the config is simple and only sets the things that are needed and differ from the defaults. The `kube-api-content-type` defaults to protobuf and I don't see a need for setting it differently in our tests. 

One can argue that it's nice to have a way to customize things, but for me it's [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)

**Comment by Jeffwan** [comment]
@mm4tt all tests pass now.

**Comment by mm4tt** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**